### PR TITLE
test: add tests for multiple transactions with fixed fee

### LINKED ISSUE #11761: Add additional suites for `FIXED_FEE` (positive outcomes)
18 test cases

### PR DESCRIPTION
**Pull request by fchitakova**
**Description**:

**Related issue(s)**:

Fixes #11761 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java [hunk 1]
```diff
@@ -1592,6 +1594,98 @@ public HapiSpec transferWithInsufficientCustomFee() {
                         .hasKnownStatus(INSUFFICIENT_SENDER_ACCOUNT_BALANCE_FOR_CUSTOM_FEE));
     }
 
+    @HapiTest
+    public HapiSpec transferMultipleTimesWithFixedFeeInHBarShouldVerifyEachTransferIsPaid() {
```
Comment (petreze) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java:1598]: you can suite name to `positiveTests()` method

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java [hunk 2]
```diff
@@ -1592,6 +1594,98 @@ public HapiSpec transferWithInsufficientCustomFee() {
...
+                        getAccountBalance(alice).hasTokenBalance(fungibleToken, 5L),
+                        getAccountBalance(alice).hasTinyBars(ONE_MILLION_HBARS - 1L),
+                        getAccountBalance(bob).hasTokenBalance(fungibleToken, 3L),
+                        getAccountBalance(bob).hasTinyBars(ONE_MILLION_HBARS - 1L),
+                        getAccountBalance(carol).hasTokenBalance(fungibleToken, 1L),
+                        getAccountBalance(carol).hasTinyBars(ONE_MILLION_HBARS - 1L));
+    }
+
+    @HapiTest
+    public HapiSpec transferMultipleTimesWithFixedFeeInCustomFungibleTokenShouldVerifyEachTransferIsPaid() {
```
Comment (petreze) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java:1639]: you can suite name to `positiveTests()` method

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java [hunk 3]
```diff
@@ -1592,6 +1594,98 @@ public HapiSpec transferWithInsufficientCustomFee() {
...
+                        cryptoTransfer(moving(5L, fungibleToken).between(alice, bob))
+                                .fee(1L),
+                        cryptoTransfer(moving(2L, fungibleToken).between(bob, carol))
+                                .fee(1L),
+                        cryptoTransfer(moving(1L, fungibleToken).between(carol, ivan))
+                                .fee(1L))
+                .then(
+                        getAccountBalance(hbarCollector).hasTinyBars(ONE_HUNDRED_HBARS + 4L),
+                        getAccountBalance(alice).hasTokenBalance(fungibleToken, 5L),
+                        getAccountBalance(alice).hasTinyBars(ONE_MILLION_HBARS - 1L),
```
Comment (petreze) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java:1631]: you can combine several checks into method chaining i.e: `getAccountBalance(alice).hasTinyBars(ONE_MILLION_HBARS - 1L).hasTokenBalance(fungibleToken, 5L)`

**_apply to all similar occurrences_**

Comment (fchitakova) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java]: @petreze , thanks, I applied the suggested changes. Please re-review.

**Review by petreze — DISMISSED** [review]
LGTM

**Review by petreze — DISMISSED** [review]

**Review by tinker-michaelj — DISMISSED** [review]
LGTM, ty @fchitakova !

**Review by petreze — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]

**Pull request closed without merging**